tests/acceptance/pageObjects/filesPage.js

```diff
@@ -261,6 +261,7 @@ module.exports = {
           .waitForElementNotVisible('@newFolderButton')
           .click('@dialogConfirmBtn')
           .waitForElementNotPresent('@dialog')
+          .waitForAjaxCallsToStartAndFinish()
```

Comment:
the `wait` is needed for the `ajax calls` started by `confirm button`

Reply:
Since the tests are slower in drone, just waiting for the upload dialog to disappear is not working here. We need to wait till the actual request that uploads the file is completed. Even when the dialog disappears it takes some time for the request to complete and the actual file to be updated in the server.